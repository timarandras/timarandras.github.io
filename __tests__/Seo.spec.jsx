import { render } from '@testing-library/react';
import { useStaticQuery } from 'gatsby';
import React from 'react';

import Seo from '../src/components/Seo';

jest.mock('gatsby');

describe('Seo', () => {
  test('Matches snapshot', () => {
    useStaticQuery.mockReturnValueOnce({ site: { siteMetadata: { description: 'Dummy site' } } });
    const { container } = render(<Seo title="dummy" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
