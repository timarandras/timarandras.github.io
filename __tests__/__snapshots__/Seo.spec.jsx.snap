// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seo Matches snapshot 1`] = `null`;
